pages/components/layout.tsx

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head'
+import Sidebar from './sidebar'
+
+export default function Layout({ children }: any) {
```

Comment:
please make use of rocket annotation for your components and functions. 

```suggestion
export const Layout = ({ children }: any) => {
```